internal/deploy/parameters.go

```diff
@@ -22,6 +22,7 @@ import (
 
 // Command line flags
 const (
+	FlagApp            = "app"
```

Comment:
I like the reuse of this variable across multiple files, but I think that this is a bit out of the scope for this file. Do you think that in the future moving all reused flags to a distinct package would be a good idea?

Reply:
Originally the app_deploy code got moved into the deploy directory, but through suggestions on the pr it got moved back to ketch/. These flags were originally private to deploy, but had to be made public because of this moving of the functionality back to ketch/.

Reply:
i tried to follow pattern for now, we can plan future refactor of this as it's out of scope a bit